packages/reassure-compare/src/output/console.ts

```diff
@@ -24,6 +27,25 @@ export function printToConsole(data: CompareResult) {
   console.log('');
 }
 
+function printMetadata(name: string, metadata?: MeasurementMetadata) {
+  if (metadata.branch && metadata.commitHash) {
+     console.log(`- ${name}: \`${metadata.branch}\` (\`${metadata.commitHash}\`)`);
```

Comment:
Seems like you have extra back quotes here.
```suggestion
     console.log(`- ${name}: ${metadata.branch} (${metadata.commitHash})`);
```